tests/tests_pytorch/utilities/test_cli.py

```diff
@@ -852,7 +852,7 @@ def after_tune(self):
 
 def test_lightning_cli_subcommands():
     subcommands = LightningCLI.subcommands()
-    trainer = Trainer()
+    trainer = Trainer(accelerator="auto")
```

Comment:
The Trainer does not run in this test, so this has no purpose

```suggestion
    trainer = Trainer()
```